clang/include/clang/CIR/Dialect/IR/CIROps.td

```diff
@@ -2834,13 +2834,14 @@ def CIR_InlineAsmOp : CIR_Op<"asm", [RecursiveMemoryEffects]> {
   let results = (outs Optional<CIR_AnyType>:$res);
 
   let arguments = (
-    ins StrAttr:$asm_string,
+    ins Variadic<AnyType>:$operands,
+        StrAttr:$asm_string,
         StrAttr:$constraints,
         AsmFlavor:$asm_flavor);  
 
   let assemblyFormat = [{
     `(`$asm_flavor`,` `{` $asm_string $constraints `}` `)` attr-dict 
-    `:` type($res)
+     operands `:` functional-type(operands, results)
```

Comment:
Please update the docs with the new added printing style.

Reply:
:( totally forgot about docs

Reply:
@bcardosolopes  updated !